tutorials/clustering.md

```diff
@@ -13,48 +12,53 @@ In this tutorial, we will show how you can create a distributed hypertable withi
 a distributed database. We will set up the distributed database with one primary access node
 and two data nodes across three distinct TimescaleDB instances.
 
-Prerequisites:
-- 3 instances of TimescaleDB installed (private BETA branch only)
-- A superuser set up on all 3 instances of TimescaleDB
-- Network connectivity between the 3 TimescaleDB instances
-- Must be on PG11
+It is assumed before starting this tutorial you will have three networked machines set up with:
+- TimescaleDB version 2.0 (beta) or greater
+- PostgresSQL 11
+
+For the sake of this tutorial we will refer to these machines as
+access.example.com, data1.example.com, data2.example.com. You will also
+need a common superuser role on each postgres instance. This tutorial
+will assume the `postgres` role for this, but you may substitute any
+role common to all machines with superuser permission.
+
+Beyond this, please ensure that you've completed the following steps.
+Some or most of these may not be necessary depending on how you
+initially set up your instances:
+- Make sure you've enabled 'max_prepared_transactions' on all the instances, this parameter is located in `postgresql.conf`. We recommend a setting of at least 100.
+- Make sure the data nodes are properly listening for network traffic. `postgresql.conf` should have a `listen_addresses` set to include the access node. If this parameter is missing or just `localhost`, change it to `access.example.com`, or the corresponding IP. You can also set it to `"*"` to not restrict where connections can originate from.
+- The superuser role, `postgres` in this example, must have an entry in the `pg_hba.conf` file for the data nodes. This should look something like: ```host    all    postgres    access.example.com    trust``` You can use any CIDR block covering the access node's IP in place of `access.example.com`. For using authentication mechanism's other than `trust`, please see the [authentication guide][data-node-authentication].
+
+After making any of these changes, please restart your Postgres instance to ensure the
+new settings take effect.
 
 ## Define the topology for your distributed database
 
-Now that you have 3 running instances of TimescaleDB, you can go ahead and set up your
-distributed database. Connect to the instance that you would like to act as the access node,
-and create a new database there:
+Now that you have 3 running instances of TimescaleDB, you can go ahead
+and set up your distributed database. Connect to `access.example.com`
+using your superuser role, and create a new database there. It's
+important that none of the instances already have a `multinode`
```

Comment:
Not very clear if the `multinode` is because there is a hard-coded limitation that disallow `multinode` or if it is the "new database" that you're referring to. Suggest "... and create a new database `multinode`. It's ...".

Reply:
done